mods/ra/maps/soviet-02b/soviet02b.lua

```diff
@@ -0,0 +1,182 @@
+EnemyBaseEntranceShroudTrigger = { CPos.New(80, 73), CPos.New(81, 73), CPos.New(82, 73), CPos.New(83, 73), CPos.New(84, 73), CPos.New(85, 73),CPos.New(86, 73), CPos.New(87, 73), CPos.New(88, 73), CPos.New(89, 73) }
+
+SendUSSRParadrops = function()
+	paraproxy1 = Actor.Create("powerproxy.paratroopers", false, { Owner = player })
+	paraproxy1.SendParatroopers(ParachuteBaseEntrance.CenterPosition, false,  Facing.North)
+	paraproxy1.Destroy()
+end
+
+SendUSSRParadropsBase = function()
+	paraproxy2 = Actor.Create("powerproxy.paratroopers2", false, { Owner = player })
+	paraproxy2.SendParatroopers(ParachuteBase1.CenterPosition, false, Facing.East)
+	paraproxy2.Destroy()
+	paraproxy3 = Actor.Create("powerproxy.paratroopers3", false, { Owner = player })
+	paraproxy3.SendParatroopers(ParachuteBase2.CenterPosition, false, Facing.East)
+	paraproxy3.Destroy()
+end
+
+Trigger.OnEnteredFootprint(BridgeShroudTrigger, function(a, id)
+	if not bridgeShroudTrigger and a.Owner == player then
+		bridgeShroudTrigger = true
+		cameraBridge = Actor.Create("camera", true, { Owner = player, Location = CameraBridge.Location })
+		Trigger.AfterDelay(DateTime.Seconds(15), function()
+			if not cameraBridge.IsDead then
+				cameraBridge.Kill()
+			end
+		end)
+	end
+end)
+
+Trigger.OnEnteredFootprint(BridgeExplosionTrigger, function(a, id)
+	if not bridgeExplosionTrigger and a.Owner == player then
+		bridgeExplosionTrigger = true
+		if not BarrelBridge.IsDead then
+			BarrelBridge.Kill()
+		end
+	end
+end)
+
+Trigger.OnEnteredFootprint(EnemyBaseEntranceShroudTrigger, function(a, id)
+	if not enemyBaseEntranceShroudTrigger and a.Owner == player then
+		enemyBaseEntranceShroudTrigger = true
+		cameraBaseEntrance = Actor.Create("camera", true, { Owner = player, Location = CameraBaseEntrance.Location })
+		Trigger.AfterDelay(DateTime.Seconds(15), function()
+			if not cameraBridge.IsDead then
+				cameraBaseEntrance.Kill()
+			end
+		end)
+	end
+end)
+
+Trigger.OnEnteredFootprint(EnemyBaseShroudTrigger, function(a, id)
+	if not enemyBaseShroudTrigger and a.Owner == player then
+		enemyBaseShroudTrigger = true
+		cameraBase1 = Actor.Create("camera", true, { Owner = player, Location = CameraBase1.Location })
+		cameraBase2 = Actor.Create("camera", true, { Owner = player, Location = CameraBase2.Location })
+		Trigger.AfterDelay(DateTime.Seconds(15), function()
+			if not cameraBase1.IsDead and not cameraBase2.IsDead then
+				cameraBase1.Kill()
+				cameraBase2.Kill()
+			end
+		end)
+	end
+end)
+
+Trigger.OnEnteredFootprint(ParachuteTrigger, function(a, id)
+	if not parachuteTrigger and a.Owner == player then
+		parachuteTrigger = true
+		SendUSSRParadrops()
+		Media.PlaySpeechNotification(player, "ReinforcementsArrived")
+	end
+end)
+
+Trigger.OnEnteredFootprint(TransportTrigger, function(a, id)
+	if not transportTrigger and a.Type == "truk" then
+		transportTrigger = true
+		if not TransportTruck.IsDead then
+			TransportTruck.Wait(DateTime.Seconds(5))
```

Comment:
Oh sorry, `Requires Traits: AttackBase, IMove`, and `truk` doesn't have an instance of `AttackBase`.